modular_citadel/code/modules/mob/cit_emotes.dm

```diff
@@ -221,7 +219,7 @@
 	emote_type = EMOTE_AUDIBLE
 
 /datum/emote/living/bark/run_emote(mob/living/user, params)
-	if(ishuman(user))
+	if(iscarbon(user) || ispAI(user))
```

Comment:
It makes no sense for non-carbons / non-pAI to be able to bark silently either. But I'm not here to tell you to add dogs to this one's `mob_type_allowed_typecache`. Or am I?